### PR TITLE
Use motor name for Joint_states_publisher

### PR DESCRIPTION
**Pull request by Simon-Steinmann**
Use the name of the motor instead the name of the positionSensor as the published joint name. This is more consistent with ROS convention

### CODE REVIEW

File: webots_ros2_core/webots_ros2_core/joint_state_publisher.py [hunk 1]
```diff
@@ -46,7 +46,8 @@ def __init__(self, robot, jointPrefix, node):
         for i in range(robot.getNumberOfDevices()):
             device = robot.getDeviceByIndex(i)
             if device.getNodeType() == Node.POSITION_SENSOR:
```
Comment (DavidMansolino) [inline webots_ros2_core/webots_ros2_core/joint_state_publisher.py:48]:
```suggestion
            if device.getNodeType() == Node.Motor:
```

Comment (Simon-Steinmann) [reply webots_ros2_core/webots_ros2_core/joint_state_publisher.py:48]: We only need the motor name, the rest still needs the sensor. So I think my solution is the simpler one. We could switch to select for motor, but then need to assign the sensor by `sensor = device.getPositionSensor()`

Comment (DavidMansolino) [reply webots_ros2_core/webots_ros2_core/joint_state_publisher.py:48]: Oh ok I understand, in that case, we should probably change the name to:
```Python
name = motor.getName() if motor is not None else name = device.getName()
```
Just to handle the case where there is no motor on this joint.

Comment (Simon-Steinmann) [reply webots_ros2_core/webots_ros2_core/joint_state_publisher.py:48]: sounds good to me, can you add the change?

**Review by DavidMansolino — APPROVED** [review]
That looks now all good, I just changed the target branch as master should work with the latest official release of Webots.

**Review by lukicdarkoo — APPROVED** [review]
Works good, thanks!

**Pull request merged**